compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -845,8 +845,8 @@ renderRow world parties NodeInfo{..} =
                     | party `S.member` niDivulgences = ("D", Just "Divulged")
                     | otherwise = ("-", Nothing)
             in
-            H.td H.! A.class_ "disclosure" $ H.div H.! A.class_ "tooltip" $ do
-                H.text label
+            H.td H.! (A.class_ (H.textValue $ T.pack $ concat $ ["disclosure"] <> [" disclosed" | isJust mbHint])) $ H.div H.! A.class_ "tooltip" $ do
```

Comment:
What is the purpose of the `disclosed` class?

Reply:
Indeed it is. It should not be. Its purpose is to scope the replacement by X to only the disclosed cells, i.e. not replace `-`. It seems like I forgot to do that. I'll add it :-)

Reply:
Fixed by
```patch
$ git add -p
diff --git a/compiler/daml-extension/src/webview.css b/compiler/daml-extension/src/webview.css
index 05d9bd785..8060f088c 100644
--- a/compiler/daml-extension/src/webview.css
+++ b/compiler/daml-extension/src/webview.css
@@ -31,10 +31,10 @@ td.disclosure {
 tr.archived td.disclosure {
   text-decoration: none;
 }
-.hidden_disclosure td.disclosure span {
+.hidden_disclosure td.disclosed.disclosure span {
   display: none;
 }
-.hidden_disclosure td.disclosure:after {
+.hidden_disclosure td.disclosed.disclosure:after {
   content: 'X';
 }
 th {
(1/1) Stage this hunk [y,n,q,a,d,s,e,?]? y
```

<img width="408" alt="Screenshot 2020-07-02 at 19 33" src="https://user-images.githubusercontent.com/45561385/86391967-ed461500-bc9a-11ea-952e-bee7f5c40cf7.png">
<img width="405" alt="Screenshot 2020-07-02 at 19 33 1" src="https://user-images.githubusercontent.com/45561385/86391981-f3d48c80-bc9a-11ea-8bae-1f94edd053a5.png">

Reply:
That looks great! One suggestion to simplify the code:
```suggestion
            H.td H.! (A.class_ $ H.textValue $ T.unwords $ "disclosure" : ["disclosed" | isJust mbHint]) $ H.div H.! A.class_ "tooltip" $ do
```
I'm a bit sad we can't do
```haskell
A.td H.! A.class_ "disclosure" H.!? (isJust mbHint, A.class_ "disclosed") $ ...
```
but unfortunately, the `blaze` library does not combine multiple `class` attributes. 😞

Reply:
To be fair to Blaze, that's really a quirk of the HTML standard: the `class` attribute looks like a string but is really a list. AFAIK it's the only one that behaves like that, other attributes don't have any "merge" semantics, so this would require a lot of special-casing.